content/blogposts/2020/our-abcs-always-be-coding-childrens-book.md

```diff
@@ -36,13 +36,15 @@ The A-Z format introduces simple concepts in a fun and easy-to-understand way so
   id="6bCO63O4swI"
 />
 
-<BlockquoteWithBorder
-  quote={`Dinosaurs, llamas, and dump trucks are cool—but so is coding. At Sourcegraph, many of us, myself included, have young children who now see us working on the computer all day. We love software development and wrote this book as a fun way to share our passion for coding with our kids.`}
-  author='Quinn Slack'
-/>
+<section className="text-center">
+  <Blockquote
+    quote="Dinosaurs, llamas, and dump trucks are cool—but so is coding. At Sourcegraph, many of us, myself included, have young children who now see us working on the computer all day. We love software development and wrote this book as a fun way to share our passion for coding with our kids."
+    author='Quinn Slack'
+  />
+</section>
```

Comment:
Did you want to use the new center prop here?

```suggestion
<Blockquote
  quote="Dinosaurs, llamas, and dump trucks are cool—but so is coding. At Sourcegraph, many of us, myself included, have young children who now see us working on the computer all day. We love software development and wrote this book as a fun way to share our passion for coding with our kids."
  author='Quinn Slack'
  center={true}
/>
```

Reply:
Yess, good call. Doing so many small changes lately to a lot of spots! Getting lost in the sauce 🙄

Reply:
Haha I feel you! No worries @st0nebraker 🙃